test/system/170-run-userns.bats

```diff
@@ -78,3 +78,20 @@ EOF
     # Then check that the main user is not mapped into the user namespace
     CONTAINERS_CONF=$PODMAN_TMPDIR/userns_auto.conf run_podman 0 run --rm $IMAGE awk '{if($2 == "0"){exit 1}}' /proc/self/uid_map /proc/self/gid_map
 }
+
+@test "podman userns=auto and secrets" {
+    if is_rootless; then
+        egrep -q "^$(id -un):" /etc/subuid || skip "no IDs allocated for current user"
+    else
+        egrep -q "^containers:" /etc/subuid || skip "no IDs allocated for user 'containers'"
+    fi
+    test_name="test_$(random_string 12)"
+    secret_file="/tmp/secret$(random_string 12)"
```

Comment:
If you use `secret_file=$PODMAN_TMPDIR/secret$(random_string 12)` you don't have to `rm -f $secret_file` at the end.